src/bot.rs

```diff
@@ -311,8 +314,15 @@ impl RcBot {
                 if let Some(ref mut message) = val.message {
                     if let Some(text) = message.text.clone() {
                         let mut content = text.split_whitespace();
-                        if let Some(cmd) = content.next() {
-                            if cmd.starts_with("/") {
+                        if let Some(command) = content.next() {
+                            if command.starts_with("/") {
+                                let cmd = {
+                                    if command.ends_with(&*self.inner.name.borrow().as_ref().unwrap().as_str()) {
```

Comment:
This will crash when the name is None in Bot. You could also make the cmd mutable and then just cut off the last part: 
``` rust
if cmd.ends_with(name) {
    cmd = cmd.rsplitn(2, '@').skip(1).next().unwrap();
}
```
(this assumes that in name is "@BotName")